src/models/stores/groups.ts

```diff
@@ -81,7 +82,11 @@ export const GroupsModel = types
           return !!group.users.find((user) => user.id === uid);
         });
       },
-
+      get groupDocuments() {
```

Comment:
```suggestion
      get groupVirtualDocument() {
```
`groupDocuments()` sounds like a function that returns an array of documents.